component/portal/src/main/java/org/exoplatform/portal/config/UserPortalConfigService.java

```diff
@@ -269,7 +269,33 @@ public UserPortalConfig getUserPortalConfig(String portalName, String accessUser
         //
         return list;
     }
-
+    /**
+     * Returns a boolean hasNav according to if the user has at least one makable navigation i.e if he
+     * belongs to at least one group
+     * @param remoteUser the user to get the makable navigations
+     * @return true or false
+     * @throws Exception any exception
+     */
+       public boolean hasMakableNavigations(String remoteUser) throws Exception{
+           Collection<Group> groups;
+           boolean hasNav;
+           if (remoteUser == null) {
+			   hasNav = false;
+			   }
+           else if (remoteUser.equals(userACL_.getSuperUser())) {
+               hasNav = true; // as the super user is member of all groups
+           } else {
+               groups = orgService_.getGroupHandler().resolveGroupByMembership(remoteUser, userACL_.getMakableMT());
+               if (groups != null) {
```

Comment:
This if/else can be replaced by :
hasNav = (groups != null && groups.size() > 0);